fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Optimizer.java

```diff
@@ -438,6 +452,11 @@ private OptExpression logicalRuleRewrite(ConnectContext connectContext,
             ruleRewriteIterative(tree, rootTaskContext, RuleSetType.SINGLE_TABLE_MV_REWRITE);
         }
 
+        if (isEnableSingleViewMvRewrite()) {
+            // view based mv rewrite for single view
+            viewBasedMvRuleRewrite(tree, rootTaskContext);
```

Comment:
yep, SINGLE_TABLE_MV_REWRITE and view based operator, now the strategy is if plan has been rewritten by SINGLE_TABLE_MV_REWRITE, it will not go single table view based mv rewrite.

Reply:
I mean Can we use view rewrite first?

Reply:
changed to do view based rewrite first then SINGLE_TABLE_MV_REWRITE